README.md

```diff
@@ -85,18 +85,35 @@ source .env
     ./scripts/init.sh onboard-mrc
     ```
 
-3. Parachain collator will be available at  port `$((9946 + $para_id))` which in default case is `11946`, while  embedded relay chain is running on port `$((9946 + $para_id + 1 ))` which in default case is `11947`.
+1. Parachain collator will be available at  port `$9944`.
 
-4. Link to parachain [dashboard](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A11946)
+1. Link to parachain [dashboard](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944)
 
-5. Off-boarding MRC from relay chain
+1. Off-boarding MRC from relay chain
 
     ```bash
     ./scripts/init.sh offboard-mrc
     ```
 
 Note: Clean up /tmp/mrc directory after off-boarding. This is required to avoid any conflicts with next onboarding. For local testing and devnet this will be ideal until runtime upgrades are implemented.
 
+### Ports
+
+- Default ports for MRC are:
+      - ```30333``` # p2p port
+      - ```9933```  # rpc port
+      - ```9944```  # ws port
+
+- Default ports for Validator Alice are:
+      - ```30334``` # p2p port
+      - ```9934```  # rpc port
+      - ```9945```  # ws port
+
+- Default ports for Validator Bob are:
+      - ```30335``` # p2p port
+      - ```9935```  # rpc port
+      - ```9946```  # ws port
+
```

Comment:
Keeping ports fixed since we will run one version of mrc on devenet so defaults works fine